MapboxNavigation/MapboxSpeechSynthesizerController.swift

```diff
@@ -0,0 +1,109 @@
+
+import AVFoundation
+import MapboxDirections
+import MapboxCoreNavigation
+import MapboxSpeech
+
+///
+open class MapboxSpeechSynthesizerController: NSObject, SpeechSynthesizerController {
```

Comment:
I think the separation of concerns would be clearer if we rename this class to SpeechSynthesizerController and rename the SpeechSynthesizerController protocol to SpeechSynthesizing. This class doesn’t need to conform to SpeechSynthesizing.

Reply:
I was thinking about this class as some kind of a wrapper or a proxy which manages multiple speech engines. At the same time it is possible to use a single speech engine directly in `RouteVoiceController` if user want's to, so `MapboxSpeechSynthesizerController` should adopt `SpeechSynthesizerController` too.

In this case, `MapboxSpeechSynthesizerController` does not have any crucial role in the mechanism and can be safely removed, or used with any custom set of speech synthesizers.

Yes, looks like naming could be better to display the idea :)